sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpanProcessor.java

```diff
@@ -31,6 +33,7 @@
 
   private final SpanExporter spanExporter;
   private final boolean sampled;
+  private final Map<CompletableResultCode, Boolean> pendingExports = new ConcurrentHashMap<>();
```

Comment:
this could be `Map<CompletableResultCode, Void>`, right?

Reply:
I had forgotten about `newSetFromMap`, oops

Reply:
For reference, `newSetFromMap` uses `Map<?, Boolean>` too - it's because `Void` can only be `null`, but Maps behave differently in their support for `null` values, in particular our best friend CHM does not support them.